rma_sale/wizard/sale_order_rma_wizard.py

```diff
@@ -143,7 +146,7 @@ def _compute_move_id(self):
         for record in self:
             if record.picking_id:
                 record.move_id = record.picking_id.move_lines.filtered(
-                    lambda r: (r.sale_line_id.product_id == record.product_id
+                    lambda r: (r.sale_line_id == record.sale_line_id and r.product_id == record.product_id
```

Comment:
This is not correct is there's no linked `sale_line_id` at all.

Reply:
Is there an example in wich that could happen?

Reply:
Returning a return will keep the sale_line_id?

Reply:
But this wizard comes from de `sale.order`, how could we be returning from a return?

Reply:
We do an RMA, return back to the customer, and a second RMA is needed inside the warranty period.

Reply:
Ok, I understand now. I'll check

Reply:
Ok, just tested it and indeed this approach fails at that point :/